code/game/mecha/equipment/weapons/weapons.dm

```diff
@@ -256,17 +416,40 @@
 	icon_state = "missile"
 	var/primed = null
 	throwforce = 15
+	var/devastation = 0
+	var/heavy_blast = 1
+	var/light_blast = 2
+	var/flash_blast = 4
+
+/obj/item/missile/proc/warhead_special(var/target)
+	explosion(target, devastation, heavy_blast, light_blast, flash_blast)

```

Comment:
Does this proc accept atoms? Thought it was only turfs?

Reply:
Warhead special for the default is a normal explosion, another type if made might accept like, doors, or something, as a special breaching explosion, or a human as a netmissile gimmick.

Reply:
I mean the `explosion` proc itself.

Reply:
The missiles did in fact explode when I tested them, so it appears so. I'll re-test.

Reply:
Yep, the explosion is made on the target atom.